src/styles/components/buttons/_buttons.scss

```diff
@@ -56,6 +63,7 @@
 
     .c-button__content {
       opacity: 0;
+      display: inherit;
```

Comment:
Y u need this?

Reply:
this fixes that weird `0.94px` issue when transitioning from an icon to a loading indicator

Reply:
don't ask me why, but it does

Reply:
Can revisit in the future - think it might have something to do with the svg itself and the viewbox maybe? approving.

Reply:
sure.  the odd part is that without this wrapper, it's fine.

text -> spinner === fine
icon by itself === fine
icon -> spinner !== fine